source/_integrations/lifx.markdown

```diff
@@ -114,6 +116,53 @@ Run an effect that does nothing, thereby stopping any other effect that might be
 | ---------------------- | ----------- |
 | `entity_id` | String or list of strings that point at `entity_id`s of lights. Use `entity_id: all` to target all.
 
+## Buttons
+
+The LIFX button platform creates two buttons for each LIFX device:
+
+| Entity ID                      | Default Status |
+| ------------------------------ | -------------- |
+| `button.{bulb_label}_identify` | Enabled        |
+| `button.{bulb_label}_restart`  | Disabled       |
```

Comment:
- Both are diagnostic entities and, therefore should just be enabled by default.
- Please don't describe entity IDs. They are not guaranteed, or stable. Instead, describe the provided entities.